src/main/java/com/softtek/academy/scrumm/web/EventController.java

```diff
@@ -0,0 +1,29 @@
+package com.softtek.academy.scrumm.web;
+
+import java.sql.SQLException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@CrossOrigin
+@RestController
+@RequestMapping
+public class EventController {
+	
+	@Autowired
+	private ExcelHandlerService excelHandlerService;
+	
+	@PostMapping(value = "/importFile", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```

Comment:
```suggestion
	@PostMapping(value = "/importfile", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```

En los url no importan las mayúsculas o minúsculas pero ¿qué opinas de ponerlo en minúscula? es más común escribirlo así

Reply:
No me gusta todo en minúsculas, pero dado que es una url creo que tienes razon sera lo mejor.